package.json

```diff
@@ -0,0 +1,86 @@
+{
+  "name": "@chainsafe/as-chacha20poly1305",
+  "version": "0.1.0",
+  "description": "An AssemblyScript implementation of chacha20poly1305",
+  "author": "ChainSafe Systems",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/ChainSafe/as-chacha20poly1305/issues"
+  },
+  "homepage": "https://github.com/ChainSafe/as-chacha20poly1305#readme",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/chainsafe/as-chacha20poly1305.git"
+  },
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "files": [
+    "lib",
+    "build"
+  ],
+  "scripts": {
+    "prebuild": "rm -rf ./dist && node -r ts-node/register ./scripts/codegen.ts",
+    "prepublish": "yarn build",
+    "build": "yarn asbuild:untouched && yarn asbuild:optimized && yarn build:lib",
+    "asbuild:untouched": "asc assembly/index.ts -b build/untouched.wasm -t build/untouched.wat --runtime none --validate --debug",
+    "asbuild:optimized": "asc assembly/index.ts -b build/optimized.wasm -t build/optimized.wat --runtime none --validate -O3z --noAssert",
+    "build:lib": "tsc -p tsconfig.build.json",
+    "build:web": "webpack --mode production --entry ./index.js --output ./dist/as-chacha20poly1305.min.js",
+    "test": "yarn run test:as-ci && yarn run test:unit",
+    "test:unit": "yarn run test:unit:node && yarn run test:unit:browser",
+    "test:unit:node": "mocha -r ts-node/register test/unit/*.test.ts",
+    "test:unit:browser": "karma start --single-run --browsers ChromeHeadless,FirefoxHeadless karma.config.js",
+    "benchmark": "node -r ts-node/register node_modules/.bin/benchmark 'test/perf/*.test.ts'",
+    "benchmark:local": "yarn benchmark --local",
+    "test:ci": "yarn test:as-ci",
+    "test:as": "asp --nortrace --verbose",
+    "test:as-ci": "asp --nortrace 2> /dev/null"
+  },
+  "devDependencies": {
```

Comment:
Are all of these dependencies necessary?

Reply:
will double check it 👍